### ISSUE TITLE
Kotlin build issues

### ISSUE THREAD

**Issue by Benji377** [Benji377/OpenMusic#261]
Kotlin is not building because it is unable to convert ArrayLists to a Cursor

**Comment by Benji377** [comment]
Here is an example of the issue:
```
C:\Users\benbe\Documents\Coding\JavaProjects\OpenMusic\app\build\tmp\kapt3\stubs\debug\com\musicplayer\openmusic\data\base\SongDao.java:19: error: Not sure how to convert a Cursor to this method's return type (java.util.ArrayList<com.musicplayer.openmusic.data.Song>).
    public abstract java.util.ArrayList<com.musicplayer.openmusic.data.Song> getAll();
                                                                             ^
```

**Comment by Benji377** [comment]
This issue should be fixed as of commit [1c83ca1](https://github.com/Benji377/OpenMusic/commit/1c83ca1208acdabc1e8b05ebe8eab67ead6bdeaa) where I changed all ArrayLists to MutableLists
